src/templates/registration/login.html

```diff
@@ -2,45 +2,67 @@
 {% load static %}
 
 {% block content %}
-    <div class="sixteen wide mobile eight wide computer centered column">
-        <form class="ui large form" method="POST">
+<div class="sixteen wide mobile six wide computer centered column">
+    <h2 class="ui blue centered header">
+        Login
+    </h2>
+    <div class="ui stacked segment">
+        <form class="ui form" method="POST">
             {% csrf_token %}
-            <div class="ui stacked segment">
-                <h2 class="ui header">
-                    Login
-                </h2>
-                <div class="ui field">
-                    <div class="ui left icon input">
-                        <i class="user icon"></i>
-                        <input type="text"
-                               name="username"
-                               placeholder="username"
-                               id="id_username"
-                               value="{{ form.username.value|default:""}}"
-                               maxlength="{{ form.username.field.max_length }}"
-                               required>
-                    </div>
+
+            <div class="field">
+                <a class="ui fluid basic button" href="{% url 'social:begin' 'chahub' %}">Log in with ChaHub</a>
+            </div>
+            <div class="ui horizontal divider" style="padding-bottom: 5px; padding-top: 5px;">
+                &nbsp;&nbsp;&nbsp;&nbsp; Or &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
ehhhhh should we not do inline styling and `&nbsp;` here?

Reply:
I couldn't get anything _but_ inline styling to work here, no idea why... I'll give it another go though. as far as `&nbsp;` goes, not sure how else to increase the gap around the 'Or'